src/mpid/ch4/netmod/ofi/ofi_types.h

```diff
@@ -235,6 +235,10 @@ typedef struct {
     void *am_send_seq_tracker;
     void *am_recv_seq_tracker;
 
+    /* active message trackers */
+    int am_inflight_inject_emus;
+    int am_inflight_rma_send_mrs;
+
```

Comment:
I guess these are always modified under lock protection? So the atomic types are no longer needed?

Reply:
Yes. Earlier I made them into atomic types so they are shared across all vnis. But moving them into per-vni is much cleaner.